_appmap/env.py

```diff
@@ -213,3 +236,7 @@ def initialize(**kwargs):
     Env.reset(**kwargs)
     logger = logging.getLogger(__name__)
     logger.info("appmap enabled: %s", Env.current.enabled)
+    if Env.current.log_file_creation_failed:
+        # Writing to stderr makes the REPL fail in vscode-python.
+        # https://github.com/microsoft/vscode-python/blob/c71c85ebf3749d5fac76899feefb21ee321a4b5b/src/client/common/process/rawProcessApis.ts#L268-L269
+        logger.info("appmap.log cannot be created")
```

Comment:
This needs to be logged at `warning` instead of `info` (because that's the level set for the `stderr` handler).

Reply:
While debugging with VSCode Python extension sources, I found out that writing to `stderr` also breaks REPL. That's why I'm using `logger.info` here.

It can be seen from these lines:
https://github.com/microsoft/vscode-python/blob/c71c85ebf3749d5fac76899feefb21ee321a4b5b/src/client/common/process/rawProcessApis.ts#L268-L269

Reply:
Well, that seems like a poor choice on their part, but I guess we're stuck with it. :|

Reply:
Yes, and unfortunately, there could be some configuration combinations with appmap environment variables that would lead to `logger.warning` calls during the initialization steps, triggering the same problem.

Reply:
One ugly hack would be to suppress them, or convert them to logger.info calls if we detect that `appmap.log` is not writable.